src/main/java/uk/gov/hmcts/juror/api/moj/enumeration/DisqualifyCode.java

```diff
@@ -12,7 +12,11 @@ public enum DisqualifyCode {
     C("C", "Conviction"),
     N("N", "Mental Capacity Act"),
     O("O", "Mental Health Act"),
-    R("R", "Residency");
+    R("R", "Residency"),
+    M("M", "Suffering From a Mental Disorder"),
+    J("J", "Involved in Justice Administration or a Member of the Clergy"),
+    E("E", "Electronic Police Check Failure"),
+    D("D", "JUDICIAL DISQUALIFICATION");
```

Comment:
Is the description for judical disqualification supposed to be all in uppercase?

Reply:
This is a direct copy from the database to ensure we render the descriptions correctly :)